examples/RunSingleModel.jl

```diff
@@ -45,6 +45,8 @@ save_summary(ECModel, output_file_combined)
 # Plot sankey plot of CO model
 plot_sankey(ECModel)
 
+business_plan_plot(ECModel)

```

Comment:
Nice! Add please a comment above that describes the content.
Please, do the same also in the doc/examples